sql-ballerina/tests/simple-params-query-test.bal

```diff
@@ -575,21 +575,21 @@ function queryTimeLongParam() {
     groups: ["query", "query-simple-params"]
 }
 function queryTimeTimeRecordParam() {
-    time:Time date = checkpanic time:parse("11:35:45", "HH:mm:ss");
+    time:TimeOfDay date = {hour: 11, minute: 35, second:45};
     TimeValue typeVal = new (date);
     ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE time_type = ${typeVal}`;
     validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
 }
 
-@test:Config {
-    groups: ["query", "query-simple-params"]
-}
-function queryTimeTimeRecordWithTimeZoneParam() {
-    time:Time date = checkpanic time:parse("2017-02-03T11:35:45", "yyyy-MM-dd'T'HH:mm:ss");
-    TimeValue typeVal = new (date);
-    ParameterizedQuery sqlQuery = `SELECT * from DateTimeTypes WHERE time_type = ${typeVal}`;
-    validateDateTimeTypesTableResult(queryMockClient(simpleParamsDb, sqlQuery));
-}
+//@test:Config {
```

Comment:
Shall we create an issue to track these tasks?

Reply:
issue is created: https://github.com/ballerina-platform/ballerina-standard-library/issues/1098